tailwind/tailwind.css

```
@@ -65,6 +65,34 @@
   border-right-color: var(--color-tooltip-bg);
 }
 
+/* Dark, richer tooltip variant (opt in via x-tooltip.theme.dark).*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)